docs/content/intro/community-roles.md

```diff
@@ -0,0 +1,109 @@
+to become a contributor. Here's what being a contributor means, and what you
+should do to become one.
+
+## What Being a Contributor Means
+
+Contributors have read access to the Athens repository on Github. This means that as a contributor, you'll have issues assigned to you and you'll be requested 
+to review pull requests (PRs) via the 
+[Github pull request review system](https://help.github.com/articles/about-pull-request-reviews/).
+
+We rely heavily on the Github PR review system, which means that if you review a PR as a contributor, you can decide 
+when that PR is ready to be merged.
+
+## How to Become a Contributor
+
+To become a contributor, the core maintainers of the project would like to see you:
+
+- Attend our development meetings regularly<sup>1</sup>
+- Comment on issues with your experiences and opinions
+- Add your comments and reviews on pull requests (anyone can do this as a community member)
+- Contribute PRs to fix issues
+- Open issues as you find them
+
+# Maintainers
+
+After you become a contributor, you'll have the opportunity to become a maintainer.
+Here's what being a maintainer means and how to become one.
+
+_Note: We sometimes refer to maintainers as "core maintainers," but they're the same thing._
+
+## What Being a Maintainer Means
+
+As a maintainer, you'll be doing the same things as a contributor with a few 
+extras:
+
+- Help organize our development meetings (i.e. help organize the agenda)
+- Promote the project and build community (e.g. present on it where possible, write about it, ...) when possible<sup>2</sup>
+- Triage issues (e.g. adding labels, promoting discussions, finalizing decisions)
+- Organize and promote PR reviews (e.g. prompting community members, contributors, and other maintainers to review)
+- Handle code of conduct issues
+
+## How to Become a Maintainer
+
+To become a maintainer, we would like you to see you be an effective contributor,
+and show that you can do some of the things maintainers do. Also, if you feel
+that this is you, please reach out to one or more of the maintainers. 
+
+# The End
+
+The above descriptions lay out roughly what each role is and how you can
+move into each of them. Folks all have different strengths, live
+in different places, and so on. We're a diverse group, and we want to keep it
+that way!
+
+So, everything in this document is a guideline, not a hard-and-fast rule.
+If you are really good at something, or can't do something else, talk to
+one of the maintainers and let us know what's up. We will accommodate
+everyone the best we can.
+
+---
+<p><i>
+    <sup>1</sup> Athens development meetings are during the day in US Pacific Time.
+    We know that this time can be problematic for some folks due to work commitments,
+    different time zones, and so on. If you can't come to meetings, that's totally ok
+    and doesn't mean you can't become a contributor! Just let one of the maintainers
+    know about it.
```

Comment:
👍

Reply:
What did you want to add here @michalpristas? To hang out in the channel during meetings if you are unable to use video/audio?

Reply:
what i meant: slack channel == Just let one of the maintainers know about it. if that someone does not know who the maintainer is